rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -138,40 +138,39 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
             y1 =  (original_image_height - crop_height) / 2;
         }
     }
-    
-   // std::cout<<"Fused Crop Decoder <x,y, w, h>: " << x1 << " " << y1 << " " << crop_width << " " << crop_height << std::endl;
+    // Handles the case where generated crop width / crop height can be greater than the max decoded width & max decoded height set by the user.
+    if(crop_width > max_decoded_width)
```

Comment:
please use std::min(crop_width, max_decoded_width) instead and similar for crop_height